edx_credentials_themes/templates/edx.org/credentials/programs/xseries/certificate.html

```diff
@@ -1,34 +1,38 @@
 {% extends 'edx.org/credentials/programs/base.html' %}
 {% load i18n %}
+{% load html %}
 
 {% block message %}
-  {# Translators: XSeries should not be translated #}
-  {% blocktrans trimmed with organization_name=program_details.organizations.0.key %}
-    You worked hard to earn your XSeries Program certificate from {{ organization_name }}.
-  {% endblocktrans %}
+  {% filter htmlescape  %}
+    {# Translators: XSeries should not be translated #}
+    {% blocktrans trimmed with organization_name=program_details.organizations.0.key %}
+      You worked hard to earn your XSeries Program certificate from {{ organization_name }}.
+    {% endblocktrans %}
+  {% endfilter %}
 {% endblock %}
 
 {% block accomplishment_summary %}
-  {# Translators: XSeries should not be translated #}
-  {% trans "has successfully completed all courses and received passing grades for an XSeries Program certificate in" %}
+  {# Translators: XSeries should not be translated, this phrase is preceded by the learner's name and followed by the course title #}
+  {% trans "has successfully completed all courses and received passing grades for an XSeries Program certificate in" as tmsg %}{{ tmsg | htmlescape }}
 {% endblock %}
 
 {% block accomplishment_stamp_title %}
   {# Translators: XSeries should not be translated #}
-  {% trans "XSeries Program certificate" %}
+  {% trans "XSeries Program certificate" as tmsg %}{{ tmsg | htmlescape }}
 {% endblock %}
 
 {% block accomplishment_metadata_title %}
   {# Translators: XSeries should not be translated #}
-  {% trans "About the edX XSeries Program certificate" %}
+  {% trans "About the edX XSeries Program certificate" as tmsg %}{{ tmsg | htmlescape }}
 {% endblock %}
 
 {% block accomplishment_metadata %}
   {# Translators: XSeries should not be translated #}
-  {% blocktrans trimmed %}
+  {% blocktrans trimmed asvar accomplishment_metadata_string %}
```

Comment:
I think you may need the `{% filter htmlescape %}` block.

Reply:
interpolate_html will escape the string.